### ISSUE TITLE
[Google Chrome] Combine search Tabs and History

### ISSUE THREAD

**Issue by testerez** [raycast/extensions#11202]
### Extension

https://www.raycast.com/Codely/google-chrome

### Description

I guess I'm not the only one to open the same tab multiple times because it's faster to create a new one than checking if I already have it opened or not. This is why I was excited when I discovered the `Search Tabs` command.
Though, I realise it's pretty common for me to search for an open tab which is not actually open yet. When this happens, this is a bit frustrating and defeats the purpose of the command (for my use case at least).
The ideal solution for me would be to show an option to open a new tab if the one I'm looking for is not already opened.
I figured a way to do this would be to combine `Search Tabs` and `Search History` in the same command.
Basically this would just show the open tabs first then history. That way, I could seamlessly open a website regardless of it being already opened in a tab or not. The would behave a bit like when you open an app after all. 

I believe this feature makes sense and would benefit many users because it unifies the behaviour between opening an app or a website.

If I missed a feature that would already cover this use case, please let me know!

### Steps To Reproduce

1. Search for an open that that you actually already closed
2. Notice that you can't find it
3. Search in history
4. You now find the closed tab
5. Open the history entry
6. Realize your experience could have been better :)

**Comment by flyyuan** [comment]
I also require it and am eager to put in the effort for PR

**Comment by flyyuan** [comment]
done #12190

**Comment by a-laughlin** [comment]
This'll be resolved by @flyyuan 's PR once it's approved.

**Comment by a-laughlin** [comment]
Created [another PR](https://github.com/raycast/extensions/pull/13836) since the previous one stalled.
